spec/src/main/resources/META-INF/smithy/bsp/extensions/scala.smithy

```diff
@@ -103,6 +103,7 @@ structure ScalacOptionsItem {
     /// the -classpath flag in the command line interface
     /// of scalac.
     @required
+    @deprecated(message: "This might be empty, use the buildTarget/jvmCompileClasspath endpoint")
```

Comment:
```suggestion
    @deprecated(message: "When the jvmCompileClasspath capability is supported, this might be empty and the buildTarget/jvmCompileClasspath endpoint should be used instead.")
```

Reply:
Done, thanks!